server/src/test/java/keywhiz/KeywhizConfigTest.java

```diff
@@ -18,6 +18,24 @@
   private static final String PROPERTY_PREFIX = "dw";
   private static final String TEST_CONFIGS_PREFIX = "configs/";
 
+  @Test
+  public void parsesRowHmacCheckDisabled() {
+    KeywhizConfig config = loadConfig("row-hmac-check-disabled.yaml");
+    assertEquals(KeywhizConfig.RowHmacCheck.DISABLED, config.getRowHmacCheck());
+  }
+
+  @Test
+  public void parsesRowHmacCheckEnforced() {
+    KeywhizConfig config = loadConfig("row-hmac-check-enforced.yaml");
+    assertEquals(KeywhizConfig.RowHmacCheck.ENFORCED, config.getRowHmacCheck());
+  }
+
+  @Test
+  public void parsesRowHmacCheckLogging() {
+    KeywhizConfig config = loadConfig("row-hmac-check-logging.yaml");
+    assertEquals(KeywhizConfig.RowHmacCheck.DISABLED_BUT_LOG, config.getRowHmacCheck());
+  }
```

Comment:
This test was initially failing and reproduces the specific error that we encountered.